main.tf

```diff
@@ -0,0 +1,120 @@
+terraform {
+
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = ">= 3.8"
+    }
+  }
+
+  backend "azurerm" {
+      resource_group_name  = "Cohort26_JeaRys_ProjectExercise"
+      storage_account_name = "jpdoapptfstate"
+      container_name       = "jptodoappcontainer"
+      key                  = "terraform.tfstate"
+  }
+
+}
+
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_resource_group" "main" {
+  name     = "Cohort26_JeaRys_ProjectExercise"
+}
+
+resource "azurerm_service_plan" "main" {
+  name                = "${var.prefix}-terraformed-asp"
+  location            = data.azurerm_resource_group.main.location
+  resource_group_name = data.azurerm_resource_group.main.name
+  os_type             = "Linux"
+  sku_name            = "B1"
+}
+
+#Cosmos DB Account
+
+resource "random_integer" "ri" {
+  min = 10000
+  max = 99999
+}
```

Comment:
Nothing wrong with this, but this is suggested in the Terraform examples so that people don't all end up picking the same name that collides; given that you use a name specific to you we could probably skip the random integer part (but we may not want to now, since that would destroy our DB!)

Reply:
Thanks Jack